dspace-spring-rest/src/main/java/org/dspace/app/rest/security/EPersonRestPermissionEvaluatorPlugin.java

```diff
@@ -70,4 +85,31 @@ public boolean hasPermission(Authentication authentication, Serializable targetI
 
         return false;
     }
+
+    @Override
+    public boolean hasPatchPermission(Authentication authentication, Serializable targetId, String targetType,
+                                      Patch patch) {
+
+        /**
+         * First verify that the user has write permission
+         * on the eperson.
+         */
+        if (!hasPermission(authentication, targetId, targetType, "WRITE")) {
+            return false;
+        }
+
+        List<Operation> operations = patch.getOperations();
+
+        /**
+         * Authenticated users are allowed to update their
+         * own password. Other operations are restricted to administrators.
+         */
+        for (Operation op: operations) {
+            if (op.getPath().contentEquals(EPersonOperationFactory.OPERATION_PASSWORD_CHANGE)) {
```

Comment:
We should be careful here. This says "if any of the operations is PASSWORD_CHANGED, allow the complete patch operation".

What it should be I think: "if any of the operations is NOT PASSWORD_CHANGED, then refuse the complete patch operation".

Reply:
Yes, I agree. Thanks!  I will have time to update the PR tomorrow.